### ISSUE TITLE
GCM deb package not installing

### ISSUE THREAD

**Issue by ErikPoppleton** [GitCredentialManager/git-credential-manager#545]
I'm trying to install GCM but it seems to be silently failing to install.  I first tried the main release (v2.0.567.18224) but encountered issue #528, so i instead downloaded the pre-release (v2.0.605.12951) which claims to have fixed this problem. However installation of that version appears to fail silently (the main release installed fine).  Here are the commands I ran:

```
$ sudo dpkg -i gcmcore-linux_amd64.2.0.605.12951.deb
    Selecting previously unselected package gcmcore.
    (Reading database ... 519027 files and directories currently installed.)
    Preparing to unpack gcmcore-linux_amd64.2.0.605.12951.deb ...
    Unpacking gcmcore (2.0.605.12951) ...
    Setting up gcmcore (2.0.605.12951) ...
$ git-credential-manager-core configure
    git-credential-manager-core: command not found
$ dpkg -L gcmcore-linux_amd64.2.0.605.12951.deb
    dpkg-query: package 'gcmcore-linux_amd64.2.0.605.12951.deb' is not installed
    Use dpkg --contents (= dpkg-deb --contents) to list archive files contents.
```

This is on Ubuntu 20.04.03

**Comment by MariaGuilarte** [comment]
I was facing this same issue, i was surprised you had commented just a few minutes ago. Before answering to you i just took a look at the path and realized the last release is being installed correctly but inside an additional subfolder called Release.

**Comment by MariaGuilarte** [comment]
I was facing almost this same issue after installing the latest pre-release (i come from #528 too)
usr/local/share/gcm-core/git-credential-manager-core: not found

i was surprised you had commented just a few minutes ago. Before answering here i just took a look at the path and realized the last release is being installed correctly but inside an additional subfolder called Release. I just copied the following files

git-credential-manager-core
libHarfBuzzSharp.so
libSkiaSharp.so
back to the parent folder
usr/local/share/gcm-core

Finally its working fine now, but surely this is not a good fix and i don't know the cons it may come with. I'm a Linux newcomer

**Comment by ldennington** [comment]
I believe this issue was fixed today in #539 for the next release. Until then, @ErikPoppleton I believe following @MariaGuilarte's example of copying files out of the `Release` folder should fix - here's what it should look like:

```text
usr
└── local
    └── share
        └── gcm-core
            ├── Atlassian.Bitbucket.UI
            ├── git-credential-manager-core
            ├── GitHub.UI
            ├── libHarfBuzzSharp.so
            ├── libSkiaSharp.so
            └── NOTICE
```

**Comment by ErikPoppleton** [comment]
Thank you @MariaGuilarte and @ldennington this fix worked!
